app/frontend/Shared/WebView.svelte

```diff
@@ -105,6 +106,16 @@
     }));
   }
 
+  function inputListener() {
+    let id = webviewE.getWebContentsId();
+    appGlobal.remoteApp.clickListener(id);
```

Comment:
Here, you could simply pass a callback function, which the backend can call. That would avoid using ipcRenderer. We are trying to avoid that IPC mechanism, in general, wherever possible.

Reply:
I tried that before but I get this error.

```
Error, in function on our side, called by remote side. Function call, args [object Object], function function () { [native code] }
TypeError: Error processing argument at index 0, conversion failure from 
    at Object.fromId (node:electron/js2c/browser_init:2:72876)
    at clickListener (file:///Users/jeremyc/Documents/GitHub/mustang/e2/out/main/index.js:126572:27)
    at JPCWebSocket.callListener (file:///Users/jeremyc/Documents/GitHub/mustang/e2/out/main/index.js:45971:18)
    at process.processTicksAndRejections (node:internal/process/task_queues:95:5)
    at async WSCall._incomingMessage (file:///Users/jeremyc/Documents/GitHub/mustang/e2/out/main/index.js:45556:20)
    at async WebSocket.<anonymous> (file:///Users/jeremyc/Documents/GitHub/mustang/e2/out/main/index.js:45650:9)
```